guacamole/src/main/java/org/apache/guacamole/rest/auth/AuthenticationService.java

```diff
@@ -105,11 +105,16 @@
      */
     private static final String IP_ADDRESS_REGEX = "(" + IPV4_ADDRESS_REGEX + "|" + IPV6_ADDRESS_REGEX + ")";
 
+     /**
```

Comment:
It looks like the start of this comment was indented one space too far. There is a standard formatting for JavaDoc comments. For example:

https://github.com/apache/guacamole-client/blob/8f12ae4fcb1c3385d2f214f0cfcaa29f5ff8bf64/guacamole/src/main/java/org/apache/guacamole/rest/auth/AuthenticationService.java#L113-L116